internal/testing/init.go

```diff
@@ -0,0 +1,89 @@
+package testing
+
+import (
+	"math/big"
+
+	"go.sia.tech/siad/modules"
+	"go.sia.tech/siad/types"
+)
+
+// TODO: This is quite the hack but until we have a better solution
```

Comment:
I do not like the `-tags=testing` flag, and am very eager to see it removed, but I feel that this is even worse. 😂 I'm ok continuing to use `-tags=testing` if we need it in order to use the `siad` packages in tests.